app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -419,13 +420,17 @@ class TabTrayView(
         view.tab_layout.isVisible = !multiselect
     }
 
-    private fun updateTabsForModeChanged() {
+    private fun updateTabsForMultiselectModeChanged(inMultiselect: Boolean) {
         view.tabsTray.apply {
             val tabs = view.context.components.core.store.state.getNormalOrPrivateTabs(
                 isPrivateModeSelected
             )
 
-            collectionsButtonAdapter.notifyItemChanged(0)
+            collectionsButtonAdapter.notifyItemChanged(
+                0,
+                if (inMultiselect) MultiselectModeChange.MULTISELECT else MultiselectModeChange.NORMAL
```

Comment:
How do you feel about `isMultiselectMode` for consistent naming?

Reply:
neutrally!